src/tribble/transform.py

```diff
@@ -28,14 +29,20 @@
 
 
 def transform(df: pandas.DataFrame) -> pandas.DataFrame:
-    df = _apply_transform_list(df, TRANSFORMERS)
-    return df.groupby('uuid').apply(_group_transform_df)
+    print("Applying " + str(len(TRANSFORMERS)) + " regular transformers:")
```

Comment:
We're in Python 3.6, so we can use f-strings instead.
`print(f'Applying {len(TRANSFORMERS)} regular transformers.')`

Reply:
TIL